gluten-data/src/main/scala/io/glutenproject/execution/GlutenHashAggregateExecTransformer.scala

```diff
@@ -127,9 +127,7 @@ case class GlutenHashAggregateExecTransformer(
           // Select sum from Velox Struct.
           expressionNodes.add(ExpressionBuilder.makeSelection(colIdx, 0))
           // Select isEmpty from Velox Struct.
-          expressionNodes.add(ExpressionBuilder
-            .makeCast(ConverterUtils.getTypeNode(BooleanType, nullable = true),
-              ExpressionBuilder.makeSelection(colIdx, 1), SQLConf.get.ansiEnabled))
```

Comment:
Also use Boolean type back here. In my local testing, it works well. Do you remember the issue to solve here?